articles/File_download.md

```diff
@@ -0,0 +1,60 @@
+# Download file from a website and upload it to Dropbox 
+
+## My very first function in powershell ~
+
+```powershell
+function Get-UrlInfo {
```

Comment:
Let's switch to a better name like `Save-Video-Wistia` perhaps?

Reply:
Do you mean renaming the file?

Reply:
Nope, renaming the function.